scripts/create_bridges.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Service to create bridges
+Before=libvirtd.service
+After=network.target
```

Comment:
I don't think we should use `network.target` here, as far as I am aware, `network.target` is vaguely defined and only used to make sure network services are properly killed before computer shutdown.  Here it should be `network-online.target` . See [here](https://www.freedesktop.org/wiki/Software/systemd/NetworkTarget/)

@henn  @SahilTikale Any corrections?